cmd/oas-cache/oas-cache.go

```diff
@@ -0,0 +1,73 @@
+// CLI utility that builds cache for OAS file to reduce init time.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/go-openapi/loads"
+	"github.com/go-openapi/spec"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/validate"
+)
+
+const help = `
+Usage:
+	oas-cache [SPEC_FILE] [CACHE_FILE]
+`
+
+func main() {
+	flagHelp := flag.Bool("help", false, "Print help message")
+	flag.Parse()
+
+	if *flagHelp {
+		fmt.Println(help)
+		os.Exit(0)
+	}
+
+	args := flag.Args()
+	if len(args) != 2 {
+		fmt.Println("Error: wrong number of arguments")
+		fmt.Println(help)
+		os.Exit(1)
+	}
+
+	specfile := os.Args[1]
+	cachefile := os.Args[2]
```

Comment:
why don't just use flag arguments like `-source openapi.yaml -destination expanded.json`?

Reply:
Because these are mandatory. It's common practice, isn't it?